cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPushStrategy.java

```diff
@@ -90,12 +90,21 @@
 
     public PushAttachmentsInline pushAttachmentsInline = PushAttachmentsInline.Small;
 
+    // request and response interceptors are only held here for convenience - they are eventually
+    // used by the underlying CouchClient instance
+
+    protected List<HttpConnectionRequestInterceptor> requestInterceptors;
+
+    protected List<HttpConnectionResponseInterceptor> responseInterceptors;
+
```

Comment:
Since we want to eventually turn on findbugs for sync-android, I think using reflection might be the way to do it, but that seems a bit wrong just to avoid a findbugs warning of dead store.